portfolio/src/main/webapp/contact.css

```diff
@@ -0,0 +1,58 @@
+html, body {
+    width: 100%;
+    height: 100%;
+    margin: 0;
+}
+
+.contact_content {
+    min-height: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: stretch;
+    background-color: darkgrey;
+}
+
+#message {
+    flex-grow: 1;
+}
+
+#message, #footer {
+    flex-shrink: 0;
+}
+
+#message {
+    align-content: center;
+    background-color: yellow;
+    width: 100%;
+}
```

Comment:
All of these stylings for the #message div... I don't see any actual divs in contact.html that have id="message".  Do you actually need this styling?

Reply:
Removed all #message from contact.css